vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleSpreadsheetLayout.java

```diff
@@ -107,83 +169,303 @@ private static class SampleRegistrationSheetBuilder {
         cell.setCellStyle(boldHeaderStyle);
-        spreadsheet.autofitColumn(columnIndex);
-        cell.setCellValue(columnHeader);
+        updatedCells.add(cell);
         columnIndex++;
-
       }
+      /*
+      fixColumnWidth(spreadsheet, header.indexOf("Species"), "Species", species);
+      fixColumnWidth(spreadsheet, header.indexOf("Specimen"), "Specimen", specimens);
+      fixColumnWidth(spreadsheet, header.indexOf("Condition"), "Condition",
+          conditionsToReplicates.keySet().stream().toList());
+*/
       spreadsheet.refreshCells(updatedCells);
     }
 
-    private void addProteomicsSheet(Spreadsheet spreadsheet) {
-      setAndStyleHeader(spreadsheet, sampleRegistrationService.retrieveProteomics());
+    private void prepareCommonSheetTasks(Spreadsheet spreadsheet) {
+      LinkedHashMap<String, List<String>> headerToPresets = new LinkedHashMap<>();
+      for(String head : header) {
+        headerToPresets.put(head, new ArrayList<>());
+      }
+      headerToPresets.put("Species", species);
+      headerToPresets.put("Specimen", specimens);
+      headerToPresets.put("Condition", conditionsToReplicates.keySet().stream().toList());
+      headerToPresets.put("Biological replicate id", getReplicateLabels());
+      prepareColumnHeaderAndWidth(spreadsheet, headerToPresets);
       spreadsheet.reload();
+      setupCommonDropDownColumns();
     }
 
-    private void addMetabolomicsSheet(Spreadsheet spreadsheet) {
-      setAndStyleHeader(spreadsheet, sampleRegistrationService.retrieveMetabolomics());
-      spreadsheet.reload();
+    private void addProteomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      prepareCommonSheetTasks(spreadsheet);
+
     }
 
-    private void addLigandomicsSheet(Spreadsheet spreadsheet) {
-      setAndStyleHeader(spreadsheet, sampleRegistrationService.retrieveLigandomics());
-      spreadsheet.reload();
+    private void addMetabolomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      prepareCommonSheetTasks(spreadsheet);
+    }
+
+    private void addLigandomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      prepareCommonSheetTasks(spreadsheet);
     }
 
-    private void addGenomicsSheet(Spreadsheet spreadsheet) {
-      setAndStyleHeader(spreadsheet, sampleRegistrationService.retrieveGenomics());
+    private void addGenomicsSheet(Spreadsheet spreadsheet, List<String> header) {
+      this.header = header;
+      LinkedHashMap<String, List<String>> headerToPresets = new LinkedHashMap<>();
+      for(String head : header) {
+        headerToPresets.put(head, new ArrayList<>());
+      }
+      headerToPresets.put("Species", species);
+      headerToPresets.put("Specimen", specimens);
+      headerToPresets.put("Condition", conditionsToReplicates.keySet().stream().toList());
+      headerToPresets.put("Analysis to be performed", Arrays.stream(SequenceAnalysisTypes
```

Comment:
NIT: Do you have time to talk about our lord and saviour `reuseable Enum Values? `